content/concepts/tokens/nftoken-batch-minting.md

```diff
@@ -0,0 +1,42 @@
+---
+html: nftoken-batch-minting.html
+parent: non-fungible-tokens.html
+blurb: Minting NFTokens in batches.
+filters:
+ - include_code
+labels:
+ - Non-fungible Tokens, NFTs
+status: not_enabled
+---
+
+# Batch minting
+
+There are two common approaches to minting NFTokens in batches: mint on demand and scripted minting.
```

Comment:
For consistency with the other docs, shouldn't it be "`NFToken` objects" rather than NFTokens? (If so, same on several other lines of this file)